src/switchroot/ostree-remount.c

```diff
@@ -46,16 +46,6 @@ main(int argc, char *argv[])
   struct stat stbuf;
   int i;
 
-  /* See comments in ostree-prepare-root.c for this.
-   *
-   * This service is triggered via
-   * ConditionKernelCommandLine=ostree
-   * but it's a lot easier for various bits of userspace to check for
-   * a file versus parsing the kernel cmdline.  So let's ensure
-   * the stamp file is created here too.
-   */
-  touch_run_ostree ();
```

Comment:
I've been trying at least a little bit to support the non-systemd case.  After this change in the "switchroot-as-pid1, !systemd" case nothing will be creating `/run/ostree-booted` right?  It should be harmless to leave this call in to ensure we create the file the same as we did previously.

Maybe just:

```
/* When systemd is in use this is normally created via the generator, but
 * we ensure it's created here as well for redundancy.
 */
  touch_run_ostree ();
```
?

Reply:
I think that makes sense... when I deleted it from there I debated whether creating `/run/ostree-booted` in many places was a sensible approach to cover corner cases. I'm certainly guilty of not thinking about !systemd...